dockerfiles/centos/centos-7/centos-7-base/Dockerfile

```diff
@@ -51,12 +51,13 @@ RUN yum -y install epel-release && \
         tmux \
         sudo \
         which && \
+    cp -af /etc/skel/ /etc/vncskel/ && \
```

Comment:
This one doesn't add fluxbox, whereas the other ones do. What is the difference?